### PR TITLE
Added Touhou Wiki to the Indie Wiki Buddy's wiki list

### PR DESCRIPTION
**Pull request by theparappanon**
I noticed TouhouWiki was missing despite it being by far the most authoritative source of Touhou content, so I decided to add it. I might add more wikis that I know of in the future, since I've noticed more than a few non-gaming wikis that seem to be missing from the extension.

### CODE REVIEW

File: data/sitesEN.json [hunk 1]
```diff
@@ -395,6 +395,17 @@
     "destination_platform": "mediawiki",
     "destination_icon": "tearsofthemis.png"
   },
+  {
+  "id": "en-touhou",
+  "origin": "Touhou Fandom Wiki",
+  "origin_base_url": "touhou.fandom.com",
+  "origin_content_path": "/wiki/",
+  "destination": "Touhou Wiki",
+  "destination_base_url": "https://en.touhouwiki.net/wiki/Touhou_Wiki",
```
Comment (KevinPayravi) [inline data/sitesEN.json:404]: Thank you for the submission! Please update this to just `en.touhouwiki.net`

**Review by KevinPayravi — CHANGES_REQUESTED** [review]

File: data/sitesEN.json (hunk 1 above)
Comment (theparappanon) [reply data/sitesEN.json]: Okay, just updated.

File: data/sitesEN.json [hunk 2]
```diff
@@ -395,6 +395,17 @@
     "destination_platform": "mediawiki",
     "destination_icon": "tearsofthemis.png"
   },
+  {
+  "id": "en-touhou",
+  "origin": "Touhou Fandom Wiki",
+  "origin_base_url": "touhou.fandom.com",
+  "origin_content_path": "/wiki/",
+  "destination": "Touhou Wiki",
+  "destination_base_url": "en.touhouwiki.net/wiki/Touhou_Wiki",
```
Comment (KevinPayravi) [inline data/sitesEN.json:404]: Just `en.touhouwiki.net`, no path please

Comment (theparappanon) [reply data/sitesEN.json]: Oh, I did not notice the path AT ALL. Just modified!

**Review by KevinPayravi — APPROVED** [review]

**Comment by theparappanon** [comment]
Also, can I join the contributors list to Indie Wiki Buddy? I plan to add more wikis to the extension in the future as well.

**Comment by KevinPayravi** [comment]
Thank you for the submission, will be included in the next release! And definitely feel free to add more, there are plenty more out there.

**Comment by KevinPayravi** [comment]
> Also, can I join the contributors list to Indie Wiki Buddy? I plan to add more wikis to the extension in the future as well.

I think so! Let me set up some branch protection rules and whatnot and get back to you.

**Pull request merged**